modules/javafx.media/src/main/native/gstreamer/3rd_party/glib/glib/gdate.c

```diff
@@ -2584,8 +2595,15 @@ win32_strftime_helper (const GDate     *d,
  *
  * Returns: number of characters written to the buffer, or 0 the buffer was too small
  */
+#ifdef GSTREAMER_LITE
+#ifndef G_OS_WIN32
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wformat-nonliteral"
+#endif // G_OS_WIN32
```

Comment:
Did you add this because it will fail on 32-bit Windows without it? This logic is otherwise untouched as part of the upgrade.

Reply:
G_OS_WIN32 defined for both 32 and 64-bit Windows. Yes, I disable it because build fails on Windows due to unknown GCC pragma.